.github/workflows/dailies.yml

```
@@ -0,0 +1,60 @@
+name: Deploy daily tests
+
+on:
+  schedule:
+    - cron: '0 12 * * *'  # 7 am EST every day
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  DailyTests:
+    uses: ./.github/workflows/testing.yml
+    with:
+      pytest_markexpr: 'not remote'
+      os_list: '["windows-2022","windows-latest","macos-13","macos-latest","ubuntu-latest"]'
+      python_list: '["3.10","3.11","3.12","3.13"]'
+    secrets:
+        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  DailyRemoteTests:
+    uses: ./.github/workflows/testing.yml
+    with:
+      pytest_markexpr: 'remote'
+      os_list: '["windows-2022","windows-latest","macos-13","macos-latest","ubuntu-latest"]'
+      python_list: '["3.10","3.11","3.12","3.13"]'
+    secrets:
+        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  NotifyOnFailure:
+    runs-on: ubuntu-latest
+    needs: [ DailyTests ]
+    if: ${{ always() && needs.DailyTests.result == 'failure' }}
```

examples on https://github.com/orgs/community/discussions/43415 have simpler
```suggestion
    if: failure()
```

is it any worse?

---

I vaguely remember this so it might be wrong but this is related to calling workflows. Chain of jobs. This condition will also be fulfilled if any of the requirements of this step failed. The default is that it will only be a local failure I think.

---

Thank you - that matches with my vague memory as well.

We will try this for now and I will remember/reference the previous way if this encounters problems